### DISCUSSION TITLE
chromium: Cannot access keepassxc-browser extension

### DISCUSSION THREAD

**Discussion by ghost** [netblue30/firejail#6220] [Q&A] [answered]
Hello,
I am trying to connect keepassxc-browser extension from chromium to keepassxc while chromium is sandboxed in firejail.
I did add the commented lines in firefox.profile in chromium.local:
```
whitelist ${RUNUSER}/kpxc_server
whitelist ${RUNUSER}/org.keepassxc.KeePassXC.BrowserServer
```
but it is not enough to make the extension working.
I did find that it works once I add:
```
ignore include disable-common.inc
```
I find this annoying as it removes a lot of sandbox measures.
I do not know what is it using in disable-common.inc to use the extension and it is a very long list of items.
Is there a not too much tough way to find what without testing them one by one?

**Comment by rusty-snake** [comment, marked as answer]
Known. See https://github.com/netblue30/firejail/discussions/5444#discussioncomment-4039950.

**Reply by ghost** [reply]
ok thanks!
It works with:
```
noblacklist ${RUNUSER}/app
```
